ssl/test/runner/ssl_transfer/test_helper.go

```diff
@@ -67,7 +70,7 @@ func NewTestHelper(test_file_path string, num_of_tests int) *TestHelper {
 func (helper *TestHelper) CanBeTransfer(test_case_name string) bool {
 	_, ok := helper.test_case_names[test_case_name]
 	if ok {
-		helper.test_case_names[test_case_name] = true
+		helper.test_case_names[test_case_name] = EMPTY
 	}
```

Comment:
You don't need this at all since the value will just be ignored

Reply:
Addressed.